### ISSUE TITLE
The focus goes to the previous page when swiping with VO

### ISSUE THREAD

**Issue by paulettefrancis** [cds-snc/covid-shield-mobile#211]
Swipe in VO.

When swiping on the "What COVID Alert does not do", the focus goes to the previous page when you get to the bottom of the page.

It should to to "Back" and then "Next" (this issue has been reported).

iPhone XR 13.5.1

**Comment by paulettefrancis** [comment]
Since I'm unable to edit this issue, please note this is happening on the set of pages as well as sub-pages.

**Comment by JuliannaR** [comment]
adressed in https://github.com/cds-snc/covid-shield-mobile/pull/367
will be part of the next build.
